crates/utils/re_span/README.md

```diff
@@ -0,0 +1,15 @@
+# re_span
+
+Part of the [`rerun`](https://github.com/rerun-io/rerun) family of crates.
+
+[![Latest version](https://img.shields.io/crates/v/re_span.svg?sepculative-link)](https://crates.io/crates/re_span?sepculative-link)
+[![Documentation](https://docs.rs/re_span/badge.svg)](https://docs.rs/re_span?sepculative-link)
+![MIT](https://img.shields.io/badge/license-MIT-blue.svg)
+![Apache](https://img.shields.io/badge/license-Apache-blue.svg)
+
+An integer range that always has a non-negative length.
+
+The standard `std::ops::Range` can have `start > end`.
```

Comment:
[troll] Though I understand the logic, I'm often confused by your insistence of using `<` (e.g. in `if 10 < my_var {...}`), and here you managed to give me pause by using `>` 😅